README.md

```diff
@@ -80,7 +82,7 @@ This section should be referred to for next steps when any indicator reports a f
 | S0011        | Determines if Pe-postgres Service is Running and Enabled on relevant components    | Check the Service can be started and enabled   `puppet resource service pe-postgres ensure=running`  examing /var/log/puppetlabs/postgresql/<postgresversion>/postgresql-<today>.log for failures                                                                                                                                                                                                                                                                           | Raise a support case quoting reference S0011 along with the log below, showing an unsuccessful startup /var/log/puppetlabs/postgresql/<postgresversion>/ postgresql-<today>.log                                                       |
 | S0012        | Determines if Puppet produced a report within the last run interval                | https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              | Raise a support case quoting reference S0012 along with the output of `puppet agent -td > debug.log 2>&1`                                                                                                                             |
 | S0013        | Determines if a catalog successfully applied on Puppet Agent Last run              | https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              | Raise a support case quoting reference S0013 along with the output of  `puppet agent -td > debug.log 2>&1`                                                                                                                            |
```

Comment:
Determines when a catalog successfully applied (Del note: It's unclear to me what "applied on the puppet agent last run" means. Would customers know what this means?) on Puppet Agent Last run 

- See: https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              
- Submit a support case.
- Reference S0013.
- Include the output of  `puppet agent -td > debug.log 2>&1`